db/seeds.rb

```diff
@@ -32,5 +44,13 @@ def make_email(data)
       }
     )
   )
+
   user.save
+
+  (0..Random.new.rand(5..20)).each do |_i|
+    user.articles.create do |new_article|
+      new_article.message = Faker::Lorem.paragraph_by_chars(number: 120, supplemental: false)
+      new_article.created_at = Faker::Date.between(from: 20.days.ago, to: Date.today)
+    end
```

Comment:
We usually provide attributes as options instead of block
```suggestion
    user.articles.create(
      message: Faker::Lorem.paragraph_by_chars(number: 120, supplemental: false),
      created_at: Faker::Date.between(from: 20.days.ago, to: Date.today)
    )
```